api/java/org/openyolo/api/KnownProviders.java

```diff
@@ -124,6 +121,27 @@ public static KnownProviders getApplicationBoundInstance(Context context) {
         return providers;
     }
 
+    /**
```

Comment:
We could avoid adding this test only methods by using a shadow.  I think it is preferable in this case due to the potential risk of these methods' usage.

For example:
public class ShadowKnownProviders {
   private static KnowProviders mImplemenation;
   public static void provideImpl() {...}
   public static KnownProviders getApplicationBoundInstance(Context context) {
       return mImplemenation;
   }
}